op-node/node/config.go

```diff
@@ -174,5 +202,11 @@ func (cfg *Config) Check() error {
 	if err := cfg.Plasma.Check(); err != nil {
 		return fmt.Errorf("plasma config error: %w", err)
 	}
+	if cfg.Driver.SequencerEnabled && cfg.Rollup.IsL2BlobTimeSet() && cfg.DACConfig == nil {
+		return fmt.Errorf("dac.urls must be set for sequencer when l2 blob time is set")
+	}
+	if (!cfg.Driver.SequencerEnabled || !cfg.Rollup.IsL2BlobTimeSet()) && cfg.DACConfig != nil {
```

Comment:
Does the full node require DACConfig to download the BLOB?

Reply:
Not need for full node, the BLOB can be regarded as out of protocol when derivation.

Reply:
I’m a bit uncertain about this assumption, as it differs from Alt-DA and implies that no roles in the protocol would be responsible for DA challenges. Could you please share the rationale behind this approach?

Reply:
For Alt-DA, they need to get the preimage from L1 commitmemt. But for BLOBs, they're not actually needed by EL in the protocol. Interested DAPPs may try to retrieve BLOBs by BLOB hash, and do BLOB challenge on L1 if they can't retrieve .

Reply:
Another way to think about it is by comparing it to the L1 4844 implementation. The L1 EL doesn’t require the BLOB data either, but it is the responsibility of the L1 CL (corresponding to the op-node in the context of L2) to ensure data availability without relying on roles outside the protocol.

Reply:
Yeah the sequencer actually ensures the BLOBs are valid, and submit to provided DA-server, but not required for normal full nodes.

Reply:
> Yeah the sequencer actually ensures the BLOBs are valid, and submits them to provided DA-server, but not required for normal full nodes.
> 
> In summary: BLOBs are in protocol for sequencing, out of protocol for both sequencer and full nodes when derivation.

I guess `out of protocol` needs more discussion.  The derivation will not check the BLOBs actual data, but will check whether the BLOBs are challenged or not on alt-DA contract.